src/components/Accordion/Accordion.ts

```diff
@@ -59,6 +66,12 @@ export default class Accordion extends HTMLElement {
 
         if (this.isClosing || !this.$root.open) {
             this.start();
+            $componentsManager.get().forEach((component) => {
+                if (component.name === 'Accordion' && component.uid !== this.uid) {
+                    const $target = component.instance as Accordion;
+                    $target.shrink();
+                }
+            });
```

Comment:
This should be removed. This logic imposes [exclusive accordions](https://developer.mozilla.org/en-US/blog/html-details-exclusive-accordions/) on all disclosures. The presence of the `name` attribute will let the browser take care of collapsing other disclosures in the same group.